src/DIRAC/ResourceStatusSystem/DB/ResourceManagementDB.py

```diff
@@ -95,7 +95,7 @@ class DowntimeCache(rmsBase):
   __table_args__ = {'mysql_engine': 'InnoDB',
                     'mysql_charset': 'utf8'}
 
-  downtimeid = Column('DowntimeID', String(64), nullable=False, primary_key=True)
+  downtimeid = Column('DowntimeID', String(127), nullable=False, primary_key=True)
```

Comment:
This will need a comment in the release notes to change the existing DB schema ?